src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubPullRequestFilter.java

```diff
@@ -26,17 +26,19 @@
 
 import hudson.Extension;
 import hudson.model.Descriptor;
+import hudson.model.TopLevelItem;
+import hudson.model.View;
 import hudson.views.ViewJobFilter;
-import jenkins.branch.Branch;
-import jenkins.branch.SimpleViewBranchFilter;
+import java.util.List;
+import jenkins.scm.api.SCMHead;
 import org.kohsuke.stapler.DataBoundConstructor;
 
 /**
  * A {@link ViewJobFilter} that matches {@link PullRequestSCMHead} based branches.
  *
  * @since FIXME
  */
-public class GitHubPullRequestFilter extends SimpleViewBranchFilter {
+public class GitHubPullRequestFilter extends ViewJobFilter {
```

Comment:
Does anything really break, or do we just get an `OldDataMonitor` report and from the user perspective the view silently goes away? If that is all, I say do it.